code/modules/mob/living/living_sentience.dm

```diff
@@ -1,3 +1,5 @@
+//WHY ISN'T THIS COMPONENT
```

Comment:
because it predates components

Reply:
no
![obraz](https://user-images.githubusercontent.com/37456678/122701037-ae62f380-d24c-11eb-89ed-a5209674652c.png)

Reply:
spawners have been in the game for ages lol